go.mod

```diff
@@ -11,6 +11,9 @@ exclude github.com/gorilla/sessions v1.2.1
 //	- sonatype-2019-0890	pkg:golang/github.com/pkg/sftp@v1.10.1
 replace github.com/spf13/cobra => github.com/spf13/cobra v1.6.1
 
+// [CVE-2023-48795] CWE-354: Improper Validation of Integrity Check Value
+replace golang.org/x/crypto => golang.org/x/crypto v0.17.0
```

Comment:
Do not use `replace` directives for upgrading vulnerabilities in production as it bypasses hash verification.

Reply:
These have now been removed